src/ObjectGraph.es6.js

```diff
@@ -181,9 +182,34 @@ ObjectGraph.prototype.rewriteName = function(name) {
 
 // Visit the prototype of o, given its dataMap.
 ObjectGraph.prototype.visitPrototype = function(o, dataMap) {
-  this.storeProto(uid.getId(o), this.visitObject(o.__proto__));
+  this.storeProto(uid.getId(o), this.visitObject(o.__proto__, {proto: true}));
 };
 
+// getProtoPropertyNames returns all inherited properties
+// that have exception values in their prototypes.
```

Comment:
Please expand on the *why*: We do this because we assume that the thrown exception was a `TypeError` indicating that the method expected an instance, not the prototype to be `this` on invocation.